src/main/resources/google_checks.xml

```diff
@@ -16,7 +16,10 @@
     Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
  -->
 
-<module name = "Checker">
+<module name="Checker">
+  <!-- IMPORTANT: Allows use of SuppressWarnings annotation -->
```

Comment:
I am not sure this comment is required. 
User need to read doc for module to understand what is it for.

Or we need to put this description to our xdoc documentation

Reply:
I've removed the comment